sdk/luascript/luascript.go

```diff
@@ -0,0 +1,64 @@
+package luascript
+
+import (
+	"context"
+
+	lua "github.com/yuin/gopher-lua"
+)
+
+// Check is a type which helps to call a lua script with variables to check something.
+// The lua script must return true/false
+type Check struct {
+	state                    *lua.LState
+	exceptionHandlerFunction *lua.LFunction
+	variables                map[string]string
+	IsError                  bool
+	Result                   *bool
```

Comment:
Why use pointer here ?
cf: https://segment.com/blog/allocation-efficiency-in-high-performance-go-services/